frontend/src/pulldasher/notifications.tsx

```diff
@@ -0,0 +1,18 @@
+import { useNotification } from "../notifications";
+import { getUser } from "../page-context";
+import { Pull } from "../pull";
+
+export function usePullNotification(pulls: Pull[], action: string) {
+  useNotification<Pull>(pulls, {
+    key: (pull: Pull) => pull.number,
+    filter: (pull: Pull) => pull.isMine() || pull.hasOutdatedSig(getUser()),
+    message(pulls) {
+      const titles = pulls.map((p) => p.title).join(", ");
+      console.log(action, titles);
```

Comment:
```suggestion
```

Reply:
Dropped in 9456617376afc5226d50d370c176e5c80b0c2528